assets/l10n/app_en.arb

```diff
@@ -194,6 +194,21 @@
       "filename": {"type": "String", "example": "file.txt"}
     }
   },
+  "messageListGroupUnknownUser": "(unknown user)",
```

Comment:
```suggestion
  "unknownUserName": "(unknown user)",
```

I think it makes sense to use the same form for this pretty consistently across the app — seems helpful for reducing confusion about whether it literally means an unknown user or the name some actual user has picked. So we can make the identifier more general.